tests/test_feedexport.py

```diff
@@ -1298,8 +1298,8 @@ def test_export_encoding(self):
         items = [dict({'foo': 'Test\xd6'})]
 
         formats = {
-            'json': '[{"foo": "Test\\u00d6"}]'.encode('utf-8'),
-            'jsonlines': '{"foo": "Test\\u00d6"}\n'.encode('utf-8'),
+            'json': '[{"foo": "TestÖ"}]'.encode('utf-8'),
+            'jsonlines': '{"foo": "TestÖ"}\n'.encode('utf-8'),
```

Comment:
```suggestion
            'json': b'[{"foo": "Test\xc3\x96"}]',
            'jsonlines': b'{"foo": "Test\xc3\x96"}\n',
```
We can use this syntax if you guys prefer. :-)